app/posts/apply-for-teacher-training/2021-07-23-end-of-cycle-2021.md

```diff
@@ -0,0 +1,226 @@
+</div>
+
+### If they carry over before Apply reopens, they cannot submit
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "carried-over-apply-not-open-yet.png"
+  },
+  caption: "A banner lets candidates know that applications are not open at the moment but that they can apply from 9am on 12 October 2021."
+}) }}
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "submit-button-disabled.png"
+  },
+  caption: "The submit button is disabled and candidates are informed that they can apply from 9am on 12 October 2021."
+}) }}
+
+### If they carry over before Find reopens, they cannot find courses
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "carried-over-find-closed.png"
+  },
+  caption: "Candidates cannot find courses if they carry over before Find has reopened."
+}) }}
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "courses-closed-review-page.png"
+  },
+  caption: "The courses closed message shows on the review page too."
+}) }}
+
+Candidates currently do not receive an email on their deadline to let them know that it's no longer possible to apply for courses starting in the 2021 to 2022 academic year but that they can continue applying. We should review this, as candidates can work on their application even if applications are not open.
+
+However, [candidates get an email letting them know that courses are open](/apply-for-teacher-training/end-of-cycle-2021/#letting-candidates-know-that-courses-are-open).
+
+## Candidates who applied in time but did not get a place
+
+### If there's still time to apply in the same recruitment cycle
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "normal-email-apply-again.png"
+  },
+  caption: "Candidates get the usual 'apply again' email inviting them to apply again in the same recruitment cycle."
+}) }}
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "application-unsuccessful-still-time-to-apply.png"
+  },
+  caption: "If they log in before the deadline to apply again, they'll see the usual 'apply again' message but with the addition of a deadline banner. They can apply again in the normal way, up until the deadline to apply again."
+}) }}
+
+### If it's too late to apply in the same recruitment cycle, they need to 'carry over' their application
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "application-unsuccessful-no-longer-possible-apply-again.png"
+  },
+  caption: "As soon as the deadline to apply again has passed, the message in the 'apply again' inset text changes - letting the candidate know they can apply for the next cycle. Clicking 'apply again' carries over their application."
```

Comment:
The hill I’m prepared to die on…

```suggestion
  caption: "As soon as the deadline to apply again has passed, the message in the 'apply again' inset text changes – letting the candidate know they can apply for the next cycle. Clicking 'apply again' carries over their application."
```

Reply:
@paulrobertlloyd did u replace the hyphen with an en rule or something?